src/lwc/geBatchGiftEntryTable/geBatchGiftEntryTable.js

```diff
@@ -219,7 +229,7 @@ export default class GeBatchGiftEntryTable extends LightningElement {
     }
 
     deleteDIRow(rowToDelete) {
-        const isRowToDelete = row => row.Id == rowToDelete.Id;
+        const isRowToDelete = row => row.Id === rowToDelete.Id;
```

Comment:
Was there an issue that prompted this change?

Reply:
To be safe when I do equality comparison in javascript I use `===` to guard against any unexpected type coercion.